### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DerGoogler**
- Added Termux reference
- Added Magisk to the readme for rooted users
- Small improvements

Let me know for any changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,29 +25,30 @@ iwr -useb https://raw.githubusercontent.com/ducaale/xh/master/install.ps1 | iex
...
+| Linux & macOS      | Nixpkgs    | `nix-env -iA nixpkgs.xh`  |
+| Linux & macOS      | Homebrew   | `brew install xh`         |
+| macOS              | MacPorts   | `sudo port install xh`    |
+| Windows            | Scoop      | `scoop install xh`        |
+| Windows            | Chocolatey | `choco install xh`        |
+
+[^1]: Make sure that you have Rust 1.54 or later installed
+[^2]: Your device needs to rooted and have [Magisk] installed. To install the module pick one of these [releases](https://github.com/Magisk-Modules-Alt-Repo/xhhttp/releases)
+[^3]: The xh package is available in Edge and will be in v3.17+. It is built with native-tls only.
+[^4]: You will need to add the apt repository from https://apt.cli.rs/
```
Comment (ducaale) [inline README.md:47]: Is there a way to have the footnotes next to the table? Otherwise, let's revert to using stars

File: README.md [hunk 2]
```diff
@@ -25,29 +25,30 @@ iwr -useb https://raw.githubusercontent.com/ducaale/xh/master/install.ps1 | iex
...
+| Debian & Ubuntu    | Apt[^4]    | `sudo apt install xh`     |
+| FreeBSD            | FreshPorts | `pkg install xh`          |
+| Linux & macOS      | Nixpkgs    | `nix-env -iA nixpkgs.xh`  |
+| Linux & macOS      | Homebrew   | `brew install xh`         |
+| macOS              | MacPorts   | `sudo port install xh`    |
+| Windows            | Scoop      | `scoop install xh`        |
+| Windows            | Chocolatey | `choco install xh`        |
+
+[^1]: Make sure that you have Rust 1.54 or later installed
+[^2]: Your device needs to rooted and have [Magisk] installed. To install the module pick one of these [releases](https://github.com/Magisk-Modules-Alt-Repo/xhhttp/releases)
```
Comment (ducaale) [inline README.md:45]: I think this part is redundant and could be removed. What do you think?

>Your device needs to rooted and have [Magisk] installed

File: README.md [hunk 3]
```diff
@@ -25,29 +25,30 @@ iwr -useb https://raw.githubusercontent.com/ducaale/xh/master/install.ps1 | iex
...
-
-\*\* The xh package is available in Edge and will be in v3.17+. It is built with native-tls only.
-
-\*\*\* You will need to add the apt repository from https://apt.cli.rs/
+| OS                 | Method     | Command                   |
+|--------------------|------------|---------------------------|
+| Any                | Cargo[^1]  | `cargo install xh`        |
+| Any                | [Huber]    | `huber install xh`        |
+| Android ([Termux]) | pkg        | `pkg install xh`          |
+| Android ([Magisk]) | App[^2]    | `magisk --install-module` |
```
Comment (ducaale) [inline README.md:33]: I haven't used Magisk before but I'm assuming it needs the path to the zip downloaded from https://github.com/Magisk-Modules-Alt-Repo/xhhttp/releases, correct?

```
magisk --install-module <path/to/module>
```

File: README.md (hunk 1 above)
Comment (DerGoogler) [reply README.md:47]: These is no way unfortunately. I'll revert that to stars..

File: README.md (hunk 3 above)
Comment (DerGoogler) [reply README.md]: yes

File: README.md [hunk 4]
```diff
@@ -25,29 +25,33 @@ iwr -useb https://raw.githubusercontent.com/ducaale/xh/master/install.ps1 | iex
...
-| Linux & macOS    | Homebrew   | `brew install xh`       |
-| macOS            | MacPorts   | `sudo port install xh`  |
-| Windows          | Scoop      | `scoop install xh`      |
-| Windows          | Chocolatey | `choco install xh`      |
+| OS                 | Method     | Command                                    |
+|--------------------|------------|--------------------------------------------|
+| Any                | Cargo\*    | `cargo install xh`                         |
+| Any                | [Huber]    | `huber install xh`                         |
+| Android ([Termux]) | pkg        | `pkg install xh`                           |
+| Android ([Magisk]) | App\*\*    | `magisk --install-module <path/to/module>` |
```
Comment (ducaale) [inline README.md:33]:
```suggestion
| Android ([Magisk]) | Magisk\*\* | `magisk --install-module <path/to/module>` |
```

File: README.md [hunk 5]
```diff
@@ -25,29 +25,33 @@ iwr -useb https://raw.githubusercontent.com/ducaale/xh/master/install.ps1 | iex
...
+| Linux & macOS      | Nixpkgs    | `nix-env -iA nixpkgs.xh`                   |
+| Linux & macOS      | Homebrew   | `brew install xh`                          |
+| macOS              | MacPorts   | `sudo port install xh`                     |
+| Windows            | Scoop      | `scoop install xh`                         |
+| Windows            | Chocolatey | `choco install xh`                         |
 
 \* Make sure that you have Rust 1.54 or later installed
 
-\*\* The xh package is available in Edge and will be in v3.17+. It is built with native-tls only.
+\*\* Your device needs to rooted and have [Magisk] installed. To install the module pick one of these [releases](https://github.com/Magisk-Modules-Alt-Repo/xhhttp/releases)
```
Comment (ducaale) [inline README.md:46]:
```suggestion
\*\* To install the module pick one of these [releases](https://github.com/Magisk-Modules-Alt-Repo/xhhttp/releases)
```

**Review by ducaale — CHANGES_REQUESTED** [review]

**Review by ducaale — APPROVED** [review]
Thanks!

**Pull request merged**